### DISCUSSION TITLE
Does `v help` command work on windows?

### DISCUSSION THREAD

**Discussion by nmsobri** [vlang/v#17856] [Questions and Answers]
As title, when i type `v help` on the terminal, i got this error:

`v help`: unknown help topic provided. Use `v help` for usage information.
Known help topics: .

**Comment by JalonSolov** [comment]
Yes, it works fine on Windows.  Make sure you have the latest version of V (cloned from the repo preferably, or from the zip file on github 2nd best).

**Reply by nmsobri** [reply]
no.. the one i download from latest release is not working... but when I build it manually, it worked just fine..

**Reply by fholmqvist** [reply]
Try running `v up`; it solved it for me (Ubuntu).

**Comment by wangzc66** [comment]
It doesn't work for me, too.

**Reply by JalonSolov** [reply]
Re-install V from these instructions:  https://github.com/vlang/v#installing-v-from-source

**Reply by wangzc66** [reply]
I downloaded the v from https://github.com/vlang/v/releases/download/weekly.2023.14/v_windows.zip.
`v up` resolved the issue.
